docs/create_new_shipment.rst

```diff
@@ -0,0 +1,41 @@
+Create New Shipment
+====================
+
+1. Create a new Sale and fill in all the details.
+   When carrier is selected, another tab named UPS appears along with other
+   tabs. 
+
+.. figure:: images/1.png
+
+2. Click on ``UPS`` and select Carrier Details.
+
+.. figure:: images/2.png
+
+Once done, click on save.
+
+3. When the products have been assigned the carriers, they are ready to be 
```

Comment:
Once the shipment is assigned, it is ready for packing. Click `make Shipment` button to process.